README.md

```diff
@@ -62,27 +62,25 @@ you can use `[headers]` directive like this.
 
 #### Callbacks
 
-`(onFileUploadFinish)="imageUploaded($event)"`. If `[url]` is specified this event is fired when component gets a response from the server, also in this case event has field `serverResponse` which contains the status code and response from the server `{status, response}`. If `[url]` is not specified it's fired immediately after an image(s) dropped into file-drop zone of choosed in file browser. So what you can do, is not specify `[url]` to handle upload yourself, for exapmple send the image into firebase storage. To get file use `event.file`.
+`(uploadFinish)="imageUploaded($event)"`. If `[url]` is specified this event is fired when component gets a response from the server, also in this case event has field `serverResponse` which contains the status code and response from the server `{status, response}`. If `[url]` is not specified it's fired immediately after an image(s) dropped into file-drop zone of choosed in file browser. So what you can do, is not specify `[url]` to handle upload yourself, for exapmple send the image into firebase storage. To get file use `event.file`.
```

Comment:
By convention I think we want to go with past tense here: `uploadFinished` - the event has already occurred.
The example function here should also by convention be suggested as `onUploadFinished($event)`

All in all:

`(uploadFinished)="onUploadFinished($event)"`